getting-started/002-search.md

```diff
@@ -24,7 +22,7 @@ Note the parameter values are set as follows:
 | `api_key` | [get yours here](https://mapzen.com/developers) |
 | `text` | ***YMCA*** |
 
-If you clicked on the query link above, you probably saw some cool **GeoJSON**, more on that later, with the following set of places in the results:
+If you clicked on the query link above, you saw some GeoJSON with the following set of places in the results:
```

Comment:
this still doesn't read well for me... how about:

> Clicking the click above will open a file containing the best matching results for the text `YMCA`, you'll notice the data is in a computer-friendly format called [GeoJSON format](http://geojson.org/), this may be hard for humans to read in some browsers.

> If you are having trouble seeing the JSON in your browser you can install a browser extension for [chrome](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc?hl=en) or [firefox](https://addons.mozilla.org/en-us/firefox/addon/jsonview/) which will make it easier to read.

> The the example above you will find the name of each matched location in a property named `'label'`: